README.md

```diff
@@ -19,123 +19,200 @@
 
-PyroEngine provides a high-level interface to use Deep learning models in production while being connected to the alert API.
+Both components can run independently or together in the same deployment.
 
-## Quick Tour
+---
 
-### Running your engine locally
+## PyroEngine: wildfire detection on edge devices
 
-You can use the library like any other python package to detect wildfires as follows:
+PyroEngine provides a high level interface to use deep learning models in production while being connected to the alert API.
+
+### Quick example
 
 ```python
 from pyroengine.core import Engine
 from PIL import Image
 
 engine = Engine()
 
-im = Image.open("path/to/your/image.jpg").convert('RGB')
+im = Image.open("path/to/your/image.jpg").convert("RGB")
+
+prediction = engine.predict(im)
+```
+
+---
+
+## PyroCamera API: unified camera control
+
+The `pyro_camera_api` package provides a REST API and a Python client to control cameras and retrieve images in a unified way.
 
-prediction = engine.predict(im) 
+The API supports multiple camera backends through a common abstraction:
+
+* `reolink` backend, for Reolink PTZ or static cameras
+* `rtsp` backend, for RTSP streams
+* `url` backend, for HTTP snapshot URLs
+* `fake` backend, for development and tests
+
+Each backend has its own class and inherits from the same base interface. The system selects the correct implementation at runtime based on the `backend` field in `credentials.json`. The API routes are the same for all camera types. PTZ cameras use pose and movement, other cameras ignore pose parameters without failing.
+
+### Simple API usage example
+
+```python
+from pyro_camera_api_client import Client
+
+client = Client("http://localhost:8000")
+
+# Move a PTZ camera to a pose
+client.move("ptz_camera_1", pose=2)
+
+# Get a snapshot as bytes
+image_bytes = client.snapshot("url_camera_1")
 ```
 
+---
+
 ## Setup
 
-Python 3.11 (or higher) and [pip](https://pip.pypa.io/en/stable/)/[conda](https://docs.conda.io/en/latest/miniconda.html) are required to install PyroEngine.
+Python 3.11 or higher and `pip` or `conda` are required.
 
 ### Developer installation
 
-If you wish to use the latest features of the project that haven't made their way to a release yet, you can install the package from source:
-
-```shell
+```bash
 git clone https://github.com/pyronear/pyro-engine.git
 pip install -e pyro-engine/.
 ```
 
-### Full docker orchestration
-
-In order to run the project, you will need to specify some information, which can be done using a `.env` file.
-This file will have to hold the following information:
-- `API_URL`: the URL of the API where to send alerts
-- `CAM_USER`: the username to access the camera
-- `CAM_PWD`: the password to access the camera
+### Environment variables
 
-So your `.env` file should look something like this:
+Deployments usually rely on a `.env` file with information such as:
 
-```
+```text
 API_URL=https://api.pyronear.org
 CAM_USER=my_dummy_login
```

Comment:
In the end, these camera credentials are only used for reolink backend/adapter right ? Hence shouldn't it be store in `credentials.json` ?

Reply:
yes why not but maybe let's change this in another pr